korebuild.json

```diff
@@ -1,6 +1,6 @@
 {
-    "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/release/2.1/tools/korebuild.schema.json",
-    "channel": "release/2.1",
+    "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/dev/tools/korebuild.schema.json",
+    "channel": "dev",
```

Comment:
?

Reply:
My bad. This PR needs to target release/2.1. I can quickly fix it.